### ISSUE TITLE
Getting error to install plugin that supports 'ruby'

### ISSUE THREAD

**Issue by sk1984** [GoDaddy-Hosting/ruby-sonar-plugin#14]
Unzipped the code and put it under extensions folder of sonar. Restarted the server and getting this error:

 Language is forced to ruby
14:41:35.163 INFO: ------------------------------------------------------------------------
14:41:35.163 INFO: EXECUTION FAILURE
14:41:35.164 INFO: ------------------------------------------------------------------------
14:41:35.164 INFO: Total time: 3.067s
14:41:35.238 INFO: Final Memory: 47M/277M
14:41:35.238 INFO: ------------------------------------------------------------------------
14:41:35.238 ERROR: Error during SonarQube Scanner execution
You must install a plugin that supports the language 'ruby'

Am I missing something? Do I need jar for this? If yes, can somebody please share the jar? 
Thanks!

**Comment by hheath** [comment]
Yep, you have to build the .jar yourself - see here: https://docs.sonarqube.org/display/DEV/Build+plugin
